src/csharp/Microsoft.Spark/Sql/ArrowUdfWrapper.DataFrame.cs

```diff
@@ -0,0 +1,463 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
What do you think of naming this file `DataFrameUdfWrapper.cs` instead of `ArrowUdfWrapper.DataFrame.cs`?